server/routes/v1/nodes.js

```diff
@@ -1,7 +1,12 @@
 const { default: axios } = require('axios');
 const log = require('../../lib/logger')({ name: 'nodes' });
 
-const DATA_URL = process.env.REACT_APP_DATA_URL;
+const ENV_NETWORK_MAP = {
+  mainnet: 'main',
+  testnet: 'test',
+  devnet: 'dev',
+  'nft-devnet': 'nft-dev',
```

Comment:
Why is `nft-devnet`'s key a string when the other ones have no quotes?

Reply:
Having a dash requires quotes in JavaScript since it is an invalid variable character.